src/VisualStudio/Core/Def/Implementation/ProjectSystem/CPS/ITempPECompiler.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.LanguageServices.ProjectSystem
+{
+    /// <summary>
+    /// Provides TempPE compiler access
```

Comment:
Can you add a comment here (probably not in a summary tag) that this is used by the project system?